CONTRIBUTING.md

```diff
@@ -21,6 +21,7 @@ Before merging:
 - Describe the change in `CHANGELOG.md`
 - Make sure `pyflakes assigner` passes with no errors/warnings
 - Check `pylint assigner` and make sure it's not too egregious. (Eventually we'll get the code to a point where there are no messages from `pylint`...)
```

Comment:
We should change this to require `pylint` to pass now that you've done all the work!!!

Reply:
There was some bug pylint claimed to find where StudentRepo.name() was being overriden by Repo.name (the attribute)... but I'm pretty sure this was a nonissue.

Reply:
Done.